### DISCUSSION TITLE
Support for Loki datasource derived fields config to open links in a new tab

### DISCUSSION THREAD

**Discussion by shaunmansell** [grafana/grafana#71023] [Data Sources]
Loki datasource derived field links always open in the same tab now and don't support a config option to set this default behaviour.

Other drilldown/cross-context links (e.g. panel datalinks) offer setting default opening behaviour to launch new tabs, and it'd be great if Loki derived field links did the same.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82595. Feel free to continue the discussion around this there. Thank you!
